contact.css

```diff
@@ -0,0 +1,110 @@
+@import "base.css";
+
+.contentDiv {
```

Comment:
This sounds like it could be a more semantic element than a `div`. Also, I would advise against using element names in class names, that way you can be a bit more flexible in case you want the element using this class not to be a `div`

Reply:
CAn you give me some more details on this issue?

Thank you